### PR TITLE
RN 4.10: Added warning about X.509 Subject Alternative Names

### PR DESCRIPTION
**Pull request by ctauchen**
From RN tracker: https://github.com/openshift/openshift-docs/issues/37586#issuecomment-1014267606

Preview: https://deploy-preview-41821--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes.html#ocp-4-10-notable-technical-changes

@s-urbaniak Can you check this to make sure I've got it right? Thanks!
@sjstout Anything you'd like to see different?

### CODE REVIEW

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -369,6 +369,19 @@ Users also have the option to use the `REGISTRY_AUTH_FILE` environment variable,
 
 // Note: use [discrete] for these sub-headings.
 
+[discrete]
+[id="ocp-4-10-TLS-subject-alternative-names-required"]
+==== TLS X.509 certificates must have Subject Alternative Names
+
+X.509 certificates must have a properly set Subject Alternative Names field.
```
Comment (nalhadef) [inline release_notes/ocp-4-10-release-notes.adoc:376]: The field name should be bold?
https://redhat-documentation.github.io/supplementary-style-guide/#release-notes

File: release_notes/ocp-4-10-release-notes.adoc [hunk 2]
```diff
@@ -369,6 +369,19 @@ Users also have the option to use the `REGISTRY_AUTH_FILE` environment variable,
...
+If you update your cluster without this, you risk breaking your cluster, rendering it inaccessible, or worse.
+
+In older versions of {product-title}, X.509 certificates worked without the Subject Alternative Names, so long as the Common Name field was set.
+link:https://docs.openshift.com/container-platform/4.6/release_notes/ocp-4-6-release-notes.html#ocp-4-6-tls-common-name[This behavior was removed in {product-title} 4.6].
+
+In some cases, certificates without Subject Alternative Names continued to work in {product-title} 4.6, 4.7, 4.8, and 4.9.
```
Comment (nalhadef) [inline release_notes/ocp-4-10-release-notes.adoc:382]: field name

File: release_notes/ocp-4-10-release-notes.adoc (hunk 1 above)
Comment (ctauchen) [reply release_notes/ocp-4-10-release-notes.adoc]: No, I don't think so. This isn't a GUI field name. It's more of a key-value pair in a text file. The name Subject Alternative Name is a common name for this, but the actual key is something like `subjectAltName`.

File: release_notes/ocp-4-10-release-notes.adoc [hunk 3]
```diff
@@ -369,6 +369,19 @@ Users also have the option to use the `REGISTRY_AUTH_FILE` environment variable,
 
 // Note: use [discrete] for these sub-headings.
 
+[discrete]
+[id="ocp-4-10-TLS-subject-alternative-names-required"]
+==== TLS X.509 certificates must have Subject Alternative Names
+
+X.509 certificates must have a properly set Subject Alternative Names field.
+If you update your cluster without this, you risk breaking your cluster, rendering it inaccessible, or worse.
```
[the remark this answers is not in the document]
Comment (ctauchen) [reply release_notes/ocp-4-10-release-notes.adoc:408]: s/or worse/and the servers will all catch fire, data centers for each distributed cluster will go offline, and YOU WILL BREAK THE CLOUD

File: release_notes/ocp-4-10-release-notes.adoc [hunk 4]
```diff
@@ -369,6 +369,19 @@ Users also have the option to use the `REGISTRY_AUTH_FILE` environment variable,
...
+
+In older versions of {product-title}, X.509 certificates worked without the Subject Alternative Names, so long as the Common Name field was set.
```
[the remark this answers is not in the document]
Comment (ctauchen) [reply release_notes/ocp-4-10-release-notes.adoc]: see above

File: release_notes/ocp-4-10-release-notes.adoc (hunk 2 above)
Comment (ctauchen) [reply release_notes/ocp-4-10-release-notes.adoc]: see above

File: release_notes/ocp-4-10-release-notes.adoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (nalhadef) [reply release_notes/ocp-4-10-release-notes.adoc:408]: I said it was HARD to imagine, not impossible :-)

[the remark this answers is not in the document]
Comment (sjstout) [reply release_notes/ocp-4-10-release-notes.adoc:408]: I'd drop the "or worse" from the release note. I get that this is an important warning, but "or worse" is pretty open-ended. Just my $.02 cents. :)

**Pull request merged**